css/addnew.css

```diff
@@ -47,6 +47,7 @@ main form div {
 
 #btn {
   padding: 5px;
+  margin-left: 140px;
 }
```

Comment:
- [ ] Great job so far 👏🏻 However you should use CSS styles to make the application match [this wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png)

| Original | Your project  |
|---------------|--------|
![image](https://user-images.githubusercontent.com/79058364/197881729-02c0950a-3a35-4433-b9da-b95a3a68c063.png) | ![Screenshot from 2022-10-25 23-05-58](https://user-images.githubusercontent.com/79058364/197882002-554ffbf1-fab2-4939-ac41-99db09177791.png)